DXRMapFixups/DeusEx/Classes/DXRFixupParis.uc

```diff
@@ -119,6 +123,13 @@ function PreFirstEntryMapFixes()
         Spawn(class'PlaceholderItem',,, vectm(-1464,-1649.6,-197)); //Bathroom stall 1
         Spawn(class'PlaceholderItem',,, vectm(-1096.7,-847,-197)); //Bathroom stall 2
         Spawn(class'PlaceholderItem',,, vectm(-2093.7,-293,-161)); //Club back room
+
+        p.x = -1821.85;
+        p.y = -351.37;
+        p.z = -207.11;
+        p.w = 200.0;
+        SetAllLampsState(true, true, false, p); // the two Lamp3s on the desks near the back exit, but not the one where the accountant is
```

Comment:
As an example of why a vector and a float radius might be better than a plane, you could just construct the vect and specify the radius in one line here, instead of setting the x/y/z/w in separate lines:

 SetAllLampsState(true, true, false, vect(-1821.85,-351.37,-207.11),200.0)

Reply:
(Unless, of course, there's a similar constructors for planes, which there might well be)

Reply:
Using a vector also allows you to do vectm, so that no special rotation needs to be done on the location manually

Reply:
It seems Plane doesn't have any constructor like that. The benefit with your suggestion is obvious, but I still dislike separating the values like that.

Reply:
I think something like this might be a nice compromise:

```
class Sphere
{
    var Vector center;
    var float radius;

    static Sphere NewSphere(float x, float y, float z, float r)
    {
        local Sphere s;
        s.center.x = x;
        s.center.y = y;
        s.center.z = z;
        s.radius = r;
        return s;
    }
}
```

Reply:
It would be a struct not a class. Classes require heap memory allocations. But we should just follow the convention of RadiusActors itself.

Reply:
I made a later suggestion on Discord:
```
function Plane NewSphere(float x, float y, float z, float r)
{
    local Plane p;

    p.x = x;
    p.y = y;
    p.z = z;
    p.w = r;

    return p;
}
```

Reply:
I saw, but I didn't like it. It's still an abuse of the Plane type, and highly unnecessary. And it also doesn't properly do coordinates for mirrored maps.

Reply:
It could easily work with mirrored maps with maybe two extra lines of code in SetAllLampsState. But we're just rehashing discord points here.

Reply:
Yes but my suggestion is 0 extra lines of code